sonar-core/src/main/java/org/sonar/core/issue/IssueFilterSerializer.java

```diff
@@ -60,7 +61,7 @@ public String serialize(Map<String, Object> map) {
         if (!valuesList.isEmpty()) {
           stringBuilder.append(key);
           stringBuilder.append(IssueFilterSerializer.KEY_VALUE_SEPARATOR);
-          for (Iterator<Object> valueListIter = valuesList.iterator(); valueListIter.hasNext(); ) {
+          for (Iterator<Object> valueListIter = valuesList.iterator(); valueListIter.hasNext();) {
             Object valueList = valueListIter.next();
             stringBuilder.append(valueList);
             if (valueListIter.hasNext()) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)Refactor this code to not nest more than 4 if/for/while/switch/try statements.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS134)